### ISSUE TITLE
Bug when used with Warptorio

### ISSUE THREAD

**Issue by Nivalux** [year6b7a/item-network-factorio-mod#28]
First of all, thank you for this mod. I really like it :)

I tried using it with Warptorio, which I expected it to might not work, since it changes the surface the platform is on.
After warping, the item tank broke and I got this exception, when I tried to open it:

The mod Item Network (0.3.3) caused a non-recoverable error.
Please report this error to the mod author.

```
Error while running event item-network::on_gui_opened (ID 88)
__item-network__/src/NetworkChest.lua:511: assertion failed!
stack traceback:
	[C]: in function 'assert'
	__item-network__/src/NetworkChest.lua:511: in function <__item-network__/src/NetworkChest.lua:498>
```

The item tank was requesting water. There were no other item tanks. (No other fluid providers or requesters)

After this crash, I tried opening an item chest, which worked strangely enough. I expected them both to break, but only the fluid one broke.

In warptorio, the lower floors stay and the items don't get moved. 
Items on the platform, that is warping from planet to planet, get affected by the warp. I don't know if it moves them or if it destroys them and tries to recreate them on the other planet.

It would be great if it could work, but otherwise I could work around the issue of not being able to use the item network on the moving platforms.

**Comment by year6b7a** [comment]
Glad you enjoy the mod!

That error means the network tank was somehow created without the mod registering it. My guess is that I need to add a new event handler to catch the new create events but more investigation is required.

**Comment by year6b7a** [comment]
I reproduced on my end and fixed in b13cd738eed011b1966ba052d9ac995841fad9b2.

**Comment by Nivalux** [comment]
Thank you for the quick fix!
